### PR TITLE
Dep Installer: update documentation

### PR DESCRIPTION
**Pull request by habibayassin**

### CODE REVIEW

**Comment by vvbandeira** [comment]
@vijayank88 FYI

File: docs/user/BuildLocally.md [hunk 1]
```diff
@@ -1,32 +1,20 @@
 # Build from sources locally
 
-## Prerequisites & Dependencies
+## Clona and Install Dependencies
```
Comment (vvbandeira) [inline docs/user/BuildLocally.md]:
```suggestion
## Clone and Install Dependencies
```

File: docs/user/BuildLocally.md [hunk 2]
```diff
@@ -1,32 +1,20 @@
...
-
--   `libffi-devel`
--   `tcl`
--   `time`
--   `pandas` Python library
-      -   Available from pip (e.g., `python3-pip`) to install pandas:
-          `pip3 install --user pandas`
-
-## Clone and Build
+The DependencyInstaller script installs all of the dependencies including OpenROAD dependencies if they are not already installed.
```
Comment (vvbandeira) [inline docs/user/BuildLocally.md]:
```suggestion
The `DependencyInstaller.sh` script installs all of the dependencies, including OpenROAD dependencies, if they are not already installed.
```

File: docs/user/BuildWithDocker.md [hunk 3]
```diff
@@ -28,6 +28,23 @@ For this method you only need to install
 
 ## Clone and Build
 
+### Build image generally
```
Comment (vvbandeira) [inline docs/user/BuildWithDocker.md]:
```suggestion
### Build Docker image
```

File: docs/user/BuildWithDocker.md [hunk 4]
```diff
@@ -28,6 +28,23 @@ For this method you only need to install
...
+
+The following instructions build the docker image with a parameterizied OS in two stages. 
+Dev stage for installing all the dependencies and packages that are needed to run OpenRoad and OpenRoad Flow Scripts.
+Build stage actually builds all of the binaries needed to run flows.
```
Comment (vvbandeira) [inline docs/user/BuildWithDocker.md]:
```suggestion
The following instructions build the docker image with a parameterized OS in two stages. 
Dev stage for installing all the dependencies and packages that are needed to run OpenROAD and OpenROAD Flow Scripts.
The build stage actually builds all of the binaries needed to run flow (i.e., `openroad` and `yosys`).
```

File: docs/user/BuildWithDocker.md [hunk 5]
```diff
@@ -28,6 +28,23 @@ For this method you only need to install
...
+git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts
+cd OpenROAD-flow-scripts
+./etc/DockerHelper.sh create -target=dev -os=$OS_NAME
+./etc/DockerHelper.sh create -target=builder -os=$OS_NAME
+```
+
+### Build image with default values
+
+The following instructions build the docker image with Centos 7 as base OS:
+
```
Comment (vvbandeira) [inline docs/user/BuildWithDocker.md]:
```suggestion
### Build image with default values

The following instructions build the docker image with Centos 7 as the base OS:

```

**Review by vvbandeira — CHANGES_REQUESTED** [review]

**Comment by vijayank88** [comment]
@vvbandeira 
Can you please check why its failing during docker build with CI?
I've restarted the CI run for this PR again.
Please merge the document update ASAP, as people still referring old docs and facing issue during installation.

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**